pyccel/parser/extend_tree.py

```diff
@@ -106,8 +106,9 @@ def insert_comments(ast, comment_lines_no, comments, else_no, attr='body', col_o
         return
     body        = getattr(ast, attr)
 
-    if attr == 'orelse':
-        assert col_offset is not None
+    # Fix necessary for python <= 3.6
+    if attr=='orelse' and isinstance(body[0], IfNode):
+        assert(col_offset is not None)
```

Comment:
Python's `assert` statement is not a function, so the added parentheses are unnecessary. I personally find them a bit confusing.

Reply:
I find it confusing without them ^^ It looks too much like python 2 syntax. I didn't realise that it wasn't a function though, I thought it was like `print` I will change them

Reply:
Fixed in 3a24360

Reply:
Thanks 😄